simtools/tests/test_db_handler.py

```diff
@@ -31,7 +31,7 @@ def test_reading_db_lst():
 def test_reading_db_mst_nc():
 
     logger.info('----Testing reading MST-NectarCam-----')
-    pars = db.getModelParameters('north-mst-nectarcam-d', 'prod4', testDataDirectory)
+    pars = db.getModelParameters('north-mst-NectarCam-D', 'prod4', testDataDirectory)
```

Comment:
Do the telescope names have to be written exactly this way (with capitalisation)?